test/spec/protractor_sync_test.ts

```diff
@@ -266,6 +266,12 @@ describe('Protractor extensions', () => {
         testArea.findVisible('.visible-element');
       }));
 
+      it('works with polledExpect regardless of element finder type', createTest(() => {
+        polledExpect(() => testArea.findElement('.visible-element').getElementFinder().getWebElement().isDisplayed()).toEqual(true);
+        polledExpect(() => testArea.findElement('.visible-element').isDisplayed()).toEqual(true);
```

Comment:
If this one was previously broken then there is something else wrong with polledExpect